integration/docker/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 1 ]]; then
+  echo 'expected one argument: "master", "worker", or "proxy"'
+  exit 1
+fi
+
+service=$1
+
+home=/opt/$(ls /opt | grep alluxio)
+cd ${home}
+
+for keyvaluepair in $(env | grep "ALLUXIO_"); do
```

Comment:
Hrmmm, maybe this is not too bad. The only confusion would be that there might be some configuration properties in `alluxio-site.properties` which are not actually properties right? The actual env variable would still work? For example, setting `ALLUXIO_CLASSPATH` would still work as expected, but there would be an extra `alluxio.classpath` property in the file which does not affect anything?